tree/20_cfe_basics/dry_run_lib.cf

```diff
@@ -82,6 +82,71 @@ bundle agent set_dry_run_mode(mode)
       "${configuration.debug} Enabling dry-run mode";
 }
 
+bundle agent dry_run_unique
```

Comment:
Could you add a comment to explain is is used as a datastore by push/pop_dry_run_mode?